src/e3/anod/qualifiers_manager.py

```diff
@@ -743,7 +743,7 @@ def build_space_name(self) -> str:
     def __error(self, msg: str) -> None:
         """Raise an error and print the helper."""
         print(self.__get_helper())
-        raise AnodError(msg)
+        raise AnodError(self.anod_instance.name + ": " + msg)
```

Comment:
Use of an f-string or (my favorite) call to `.format()` would prevent strings arithmetics.
Something like:

- `f"{self.anod_instance.name}: {msg}"`
- or `"{0}: {1}".format(self.anod_instance.name, msg)`

I guess that a dedicated test (or modification of an existing test) would be good too.